src/test/java/app/komunumo/util/ImageUtilIT.java

```diff
@@ -41,55 +41,60 @@ class ImageUtilIT extends IntegrationTest {
     @Autowired
     private @NotNull ImageService imageService;
 
+    private static String getSubFolder(final @NotNull UUID imageId) {
+        final String id = imageId.toString();
+        final String prefix1 = id.substring(0, 2);
+        final String prefix2 = id.substring(2, 4);
+        return prefix1 + java.io.File.separator + prefix2;
+    }
+
     @Test
     void resolveImageUrl() {
-        assertThat(ImageUtil.resolveImageUrl(null)).isNull();
+        // null should give placeholder now
+        assertThat(ImageUtil.resolveImageUrl(null)).isEqualTo("/images/komunumo.svg");
```

Comment:
This is not the placeholder. We can't set the placeholder in this method because the placeholder URL contains the width and size. The placeholder is dynamically created using the `PlaceholderImageGenerator`.